.github/workflows/customer-managed-deployment-agent-cli.yml

```diff
@@ -0,0 +1,105 @@
+env:
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+name: customer-managed-deployment-agent cli docs build
+on:
+  repository_dispatch:
+    types:
+      - customer-managed-deployment-agent
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    needs: build-customer-managed-deployment-agent-cli-docs
+    steps:
+      - name: checkout docs repo
+        uses: actions/checkout@v2
+      - name: set the customer-managed-deployment-agent version
+        run: |
+          echo "CMDA_VERSION=${{ github.event.client_payload.ref }}" >> $GITHUB_ENV
+      - name: pull-request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "customer-managed-deployment-agent/${{ github.run_id }}-${{ github.run_number }}"
+          destination_branch: "master"
+          pr_title: "Regen docs customer-managed-deployment-agent@${{ env.CMDA_VERSION }}"
+          pr_body: "Automated PR"
+          pr_label: "automation/customer-managed-deployment-agent-cli-docs,automation/merge"
```

Comment:
I _think_ this may fail if that label `automation/customer-managed-deployment-agent-cli-docs` hasn't been created yet.

Reply:
Thanks -- I just manually created the label, and discovered that it was too long for GH. Pushed another commit to shorten it within the allowed character limit.